mm2src/coins/lightning/ln_platform.rs

```diff
@@ -20,14 +22,20 @@ use lightning::chain::{chaininterface::{BroadcasterInterface, ConfirmationTarget
                        Confirm, Filter, WatchedOutput};
 use rpc::v1::types::{Bytes as BytesJson, H256 as H256Json};
 use spv_validation::spv_proof::TRY_SPV_PROOF_INTERVAL;
-use std::cmp;
 use std::convert::{TryFrom, TryInto};
-use std::sync::atomic::{AtomicU64, Ordering as AtomicOrdering};
+use std::sync::atomic::{AtomicU64, Ordering as AtomicOrdering, Ordering};
 
 const CHECK_FOR_NEW_BEST_BLOCK_INTERVAL: f64 = 60.;
-const MIN_ALLOWED_FEE_PER_1000_WEIGHT: u32 = 253;
 const TRY_LOOP_INTERVAL: f64 = 60.;
 
+static DEFAULT_BACKGROUND_FEES_PER_VB: AtomicU64 = AtomicU64::new(1012);
+static DEFAULT_NORMAL_FEES_PER_VB: AtomicU64 = AtomicU64::new(8000);
+static DEFAULT_HIGH_PRIORITY_FEES_PER_VB: AtomicU64 = AtomicU64::new(20000);
+
+fn set_default_background_fees(fee: u64) { DEFAULT_BACKGROUND_FEES_PER_VB.store(fee, Ordering::Relaxed); }
+fn set_default_normal_fees(fee: u64) { DEFAULT_BACKGROUND_FEES_PER_VB.store(fee, Ordering::Relaxed); }
+fn set_default_high_priority_fees(fee: u64) { DEFAULT_BACKGROUND_FEES_PER_VB.store(fee, Ordering::Relaxed); }
```

Comment:
Is it possible to enable two lightning coins at once? For example, one is based on BTC and one is based on tBTC?
If so, these global variables will be invalid for one of the coins.

Also we're still trying to design HD wallet functionality, and probably we'll allow to run multiple mm2 instances at once. For mobile app and Web app these global variables will be shared that is also unacceptable.

Could you please consider moving them to a context?

Reply:
Done